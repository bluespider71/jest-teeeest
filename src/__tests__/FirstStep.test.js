import renderer from 'react-test-renderer';
import FirstStep from '../components/firstStep/firstStep'
it('renders correctly', () => {
    const tree = renderer
      .create(<FirstStep page="https://Facebook.com">Facebook</FirstStep>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});