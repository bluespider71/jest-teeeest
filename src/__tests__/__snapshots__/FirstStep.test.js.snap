// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=""
>
  <p
    className="type-text"
  >
     
    Which Type of Rule do you want to create? 
     
  </p>
  <div
    className="group-buttons"
  >
    <div
      className="first-group-buttons"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium icon-buttons first-button css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <img
          src="first-icon.png"
        />
        <p
          className="button-texts"
        >
           Union 
        </p>
        <img
          className="button-imgs-first"
          src="required.png"
        />
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium icon-buttons second-button css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <img
          src="second-icon.png"
        />
        <p
          className="button-texts"
        >
           Intersection 
        </p>
        <img
          className="button-imgs-second"
          src="required.png"
        />
      </button>
    </div>
    <div
      className="second-group-buttons"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium icon-buttons css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <img
          src="third-icon.png"
        />
        <p
          className="button-texts"
        >
           Complement 
        </p>
        <img
          className="button-imgs-third"
          src="required.png"
        />
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium icon-buttons fourth-button css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <img
          src="fourth-icon.png"
        />
        <p
          className="button-texts"
        >
           Difference 
        </p>
        <img
          className="button-imgs-fourth"
          src="required.png"
        />
      </button>
    </div>
  </div>
  <div />
</div>
`;
